lib/theme_check/liquid_node.rb

```diff
@@ -73,6 +73,21 @@ def outer_markup
       end
     end
 
+    def outer_markup_range
+      return if variable_lookup?
+      if tag? && block?
+        start_index = block_start_start_index
+        end_index = find_block_delimiter(block_start_end_index + inner_markup.size)&.end(0)
+      elsif tag?
+        start_index = block_start_start_index
+        end_index = block_start_end_index + 1
+      else
+        start_index = outer_markup_start_index
+        end_index = outer_markup_end_index + 1
+      end
+      start_index...end_index
```

Comment:
start_index..end_index would make it so you don't need the + 1.

The trick I use to remember the difference between `..` and `...` is that the last `.` "pushes" the last number out of the range.

So (0..5).to_a = [0, 1, 2, 3, 4, 5]
And (0...5).to_a = [0, 1, 2, 3, 4]

So in this case, since you _want_ the last index, start..end_index should do.